src/backend/aspen/api/schemas/usergroup.py

```diff
@@ -43,6 +43,7 @@ class UserUpdateRequest(BaseRequest):
 class UserMeResponse(UserBaseResponse):
     split_id: str
     group: GroupResponse
+    group_admin: bool
```

Comment:
Adds group admin flag to the /users/me response @mmmmmmaya

Reply:
Will this need updates when we get to multigroup, or will this stay the same structure and just update the value depending on which group is shown?

Reply:
hehe, I think that's a much bigger conversation -- let's talk next week!